### DISCUSSION TITLE
ESP32's family Memory Map 101 · Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#112] [General]
# ESP32's family Memory Map 101 · Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/esp32-memory-map-101/

**Comment by FBSeletronica** [comment]
Thank you very much, Marek, for sharing this excellent material. I will share it with my students and colleagues and use it whenever I need to reference the memory map of the ESP32’s series of SoCs.
